### PR TITLE
ui(dm): spec v3 updates

### PR DESCRIPTION
**Pull request by zoubingwu**
### What problem does this PR solve?

- Show pre-check warning when create/update task
- Start task with parameters

Issue Number: ref #5442 

#### Tests 

 - Unit test
 - Integration test

### Release note 

```release-note
None
```

### CODE REVIEW

File: dm/ui/src/pages/migration/task/index.tsx [hunk 1]
```diff
@@ -441,6 +481,18 @@ const TaskList: React.FC = () => {
...
+  }, [startTaskResult.status])
+
+  useEffect(() => {
+    showMessage(stopTaskResult)
+  }, [stopTaskResult.status])
+
+  useEffect(() => {
+    showMessage(deleteTaskResult)
+  }, [deleteTaskResult.status])
+
```
Comment (baurine) [inline dm/ui/src/pages/migration/task/index.tsx:495]: I doubt these 3 `useEffect` will cause hook dependency lint warning.

File: dm/ui/src/pages/migration/task/index.tsx [hunk 2]
```diff
@@ -459,6 +511,12 @@ const TaskList: React.FC = () => {
     refetchCurrentTaskStatus()
   }, [data])
 
+  useEffect(() => {
+    if (!isCreateTaskModalVisible) {
+      form.resetFields()
+    }
+  }, [isStartTaskModalVisible])
```
Comment (baurine) [inline dm/ui/src/pages/migration/task/index.tsx:518]: Is it a typo? the hook dependency is `isStartTaskModalVisible` but body is `isCreateTaskModalVisible`.

**Review by baurine — APPROVED** [review]
lgtm

File: dm/ui/src/models/task.ts [hunk 3]
```diff
@@ -166,8 +172,12 @@ export type DmapiStartTaskApiArg = {
   taskName: string
   startTaskRequest?: {
     remove_meta?: boolean
-    safe_mode_time_duration?: any
+    safe_mode_time_duration?: string
     source_name_list?: string[]
+
+    /**
+     * RFC3339 format date string
```
Comment (WizardXiao) [inline dm/ui/src/models/task.ts:179]: Maybe we need to record format is like `2006-01-02T15:04:05` or `2006-01-02T15:04:05` and will give UTC time value to backend.

**Review by WizardXiao — COMMENTED** [review]
i didn't find any obvious problems，just some notes need to adjust.

**Review by WizardXiao — APPROVED** [review]

**Review by lichunzhu — APPROVED** [review]
LGTM

File: dm/ui/src/models/task.ts (hunk 3 above)
Comment (zoubingwu) [reply dm/ui/src/models/task.ts:179]: sure

File: dm/ui/src/pages/migration/task/index.tsx (hunk 2 above)
Comment (zoubingwu) [reply dm/ui/src/pages/migration/task/index.tsx:518]: yeah, typo

**Pull request merged**